script.module.dropbox/addon.xml

```diff
@@ -1,33 +1,36 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.dropbox"
        name="Dropbox API"
-       version="8.4.1"
+       version="8.4.2"
        provider-name="micah">
```

Comment:
Official author of the Dropbox module is `Dropbox`. https://github.com/dropbox/dropbox-sdk-python/blob/v8.4.1/setup.py#L59.